db/migrate/20170421154535_create_default_price_groups_for_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDefaultPriceGroupsForUsers < ActiveRecord::Migration
+
+  def up
+    User.find_each(&:create_default_price_group!)
```

Comment:
We should probably remove any existing base/external memberships. I think it only happens in open, but after running the migrations, I have uses who have both internal and external groups.